src/components/chat/answer/chatActions.tsx

```diff
@@ -4,16 +4,16 @@ import React from 'react';
 import { useAIState } from 'ai/rsc';
 import { usePathname } from 'next/navigation';
 
+import FeedbackBar from '@/components/chat/answer/feedbackBar';
 import { AI } from '@/lib/actions';
 import gtagEvent from '@/lib/gtag';
 import { Feedback } from '@/models/chat';
 import { saveReaction } from '@/services/historyService';
 
-import Share from '../share/share';
-import CopyToClipboardButton from '../ui/copyToClipboardButton';
+import CopyToClipboardButton from '../../ui/copyToClipboardButton';
```

Comment:
i'm ok with this but should we go all in and `@/components/ui/copy....` just use the magic path most places if there is more than one level diff?